plugin/semisync/semisync_slave_plugin.cc

```diff
@@ -130,6 +131,14 @@ static int repl_semi_slave_read_event(Binlog_relay_IO_param *,
 static int repl_semi_slave_queue_event(Binlog_relay_IO_param *param,
                                        const char *, unsigned long, uint32) {
   if (rpl_semi_sync_slave_status && semi_sync_need_reply) {
+    DBUG_EXECUTE_IF("before_semi_sync_reply", {
+      static constexpr char act[] =
+          "now signal reached "
```

Comment:
I would add more unique names instead of `reached` and `continue` to avoid possible collisions.